docs/tr/docs/python-types.md

```diff
@@ -144,58 +140,94 @@ Yalnızca `str` değil, tüm standart Python tiplerinin bildirebilirsiniz.
+Python geliştikçe, **yeni sürümler** bu `belirteç` (tip eki) açıklamaları için gelişmiş destekle birlikte gelir ve çoğu durumda tür ek açıklamalarını bildirmek için yazım modülünü içe aktarmanıza ve kullanmanıza bile gerek kalmaz.
 
-Bu tipleri ve dahili tpileri bildirmek için standart Python modülünü "typing" kullanabilirsiniz.
+Projeniz için Python'un daha yeni bir sürümünü seçebilir ve böylece ekstra basitleştirilmiş özelliklerden yararlanabilirsiniz.
 
-Bu tür tip belirteçlerini desteklemek için özel olarak mevcuttur.
+Tüm dokümanlarda Python'un her sürümüyle uyumlu örnekler bulabilirsiniz (eğer bir farklılık oluştuysa).
 
-#### `List`
+Örneğin "**Python 3.6+**" Python 3.6 veya üzeri (3.7, 3.8, 3.9, 3.10, vb. dahil) ile uyumlu olduğu anlamına gelir. Ve "**Python 3.9+**" Python 3.9 veya üzeri (3.10 vb. dahil) ile uyumlu olduğu anlamına gelir.
+
+**Python'un en son sürümlerini** kullanayorsanız, en son sürüm için oluşturulan örnekleri kullanın. Bunlar **en iyi ve en basit sözdizimine** sahip olacaktır, örneğin, "**Python 3.10+**".
+
+#### `List` (Liste)
 
 Örneğin `str` değerlerden oluşan bir `list` tanımlayalım.
 
-From `typing`, import `List` (büyük harf olan `L` ile):
+=== "Python 3.9+"
 
-```Python hl_lines="1"
-{!../../../docs_src/python_types/tutorial006.py!}
-```
+    Declare the variable, with the same colon (`:`) syntax.
 
-Değişkenin tipini yine iki nokta üstüste (`:`) ile belirleyin.
+    As the type, put `list`.
 
-tip olarak `List` kullanın.
+    As the list is a type that contains some internal types, you put them in square brackets:
 
-Liste, bazı dahili tipleri içeren bir tür olduğundan, bunları köşeli parantez içine alırsınız:
+    ```Python hl_lines="1"
+    {!> ../../../docs_src/python_types/tutorial006_py39.py!}
+    ```
 
-```Python hl_lines="4"
-{!../../../docs_src/python_types/tutorial006.py!}
-```
+=== "Python 3.6+"
+
+    From `typing`, import `List` (büyük harf olan `L` ile):
+
+    ``` Python hl_lines="1"
+    {!> ../../../docs_src/python_types/tutorial006.py!}
+    ```
+
+    Değişkenin tipini yine iki nokta üstüste (`:`) ile belirleyin ve tip olarak `List` kullanın.
+
+    Liste, bazı dahili tipleri içeren bir tür olduğundan, bunları köşeli parantez içine alırsınız:
+
+    ```Python hl_lines="4"
+    {!> ../../../docs_src/python_types/tutorial006.py!}
 
 !!! ipucu
     Köşeli parantez içindeki bu dahili tiplere "tip parametreleri" denir.
 
-    Bu durumda `str`, `List`e iletilen tür parametresidir.
+    Bu durumda `str`, `List`e iletilen tür parametresidir (veya Python 3.9+ için `list`)
 
 Bunun anlamı şudur: "`items` değişkeni bir `list`tir ve bu listedeki öğelerin her biri bir `str`dir".
 
-Bunu yaparak, düzenleyicinizin listedeki öğeleri işlerken bile destek sağlamasını sağlayabilirsiniz:
+!!! tip
+    Python 3.9 veya daha üstünü kullanıyorsanız, `List`i `typing`den import etmek zorunda değilsiniz, bunun yerine aynı normal `list` türünü kullanabilirsiniz.
```

Comment:
`... import etmek...` => `...modülünden projenize dahil etmek...` veya `...modülünden import etmek`

`...aynı normal...` demek doğru mu? `...her zamanki...` demek çeviriye daha uygun oluyor. 

`import etmek` için ne kullanmalıyız?